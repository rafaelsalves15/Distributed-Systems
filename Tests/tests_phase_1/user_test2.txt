createAccount A bob
transferTo A alice bob 100
balance A alice
balance A bob
exit
