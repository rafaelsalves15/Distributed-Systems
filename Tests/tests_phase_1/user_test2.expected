UserClientMain
Received 2 arguments
arg[0] = localhost
arg[1] = 2001
> OK
> OK
> OK
50
> OK
100
